epub34/overview/index.html

```diff
@@ -728,6 +746,20 @@ <h3>EPUB 3.3: 2022</h3>
 
 				<p>The separate <a data-cite="epub-33#change-log">section</a> in [[epub-33]] provides a more detailed
 					overview of the changes.</p>
+
+				<p>The W3C Recommendation was published in 2023, with a minor update in 2025.</p>
+			</section>
+			<section id="epub34">
```

Comment:
I'm not sure about adding the recap in advance of work really beginning. This could easily get forgotten. What about putting the paragraph inside an editor's note so that it's more likely to get flagged as needing finalizing later?

Reply:
You are right. I have put the whole section as an "ednote"